pom.xml

```diff
@@ -16,7 +16,7 @@
         <dependency>
             <groupId>io.tiledb</groupId>
             <artifactId>tiledb-java</artifactId>
-            <version>0.3.2</version>
```

Comment:
I believe this should be put back to 0.3.2?

Reply:
Correct, I reverted that back to 0.3.2.